ui/ducks/send/send.js

```diff
@@ -1267,12 +1268,9 @@ export function useMyAccountsForRecipientSearch() {
  * @returns {void}
  */
 export function updateRecipient({ address, nickname }) {
-  return async (dispatch, getState) => {
+  return async (dispatch) => {
     await dispatch(actions.updateRecipient({ address, nickname }));
-    const state = getState();
-    if (state.send.asset.type === ASSET_TYPES.TOKEN) {
-      await dispatch(computeEstimatedGasLimit());
-    }
+    await dispatch(computeEstimatedGasLimit());
```

Comment:
This will not result in over calling eth_gasEstimate because we will return GAS_LIMITS.SIMPLE early if its a eth tx on a L1 network?

Reply:
correct